pandas/io/parquet.py

```diff
@@ -18,20 +18,23 @@ def get_engine(engine: str) -> "BaseImpl":
 
     if engine == "auto":
         # try engines in this order
-        try:
-            return PyArrowImpl()
-        except ImportError:
-            pass
+        engine_impls = [PyArrowImpl, FastParquetImpl]
 
-        try:
-            return FastParquetImpl()
-        except ImportError:
-            pass
+        error_msgs = []
+        for eimpl in engine_impls:
```

Comment:
May be use `engine_classes` and `engine_class`? Personally I find these abbreviations a bit confusing and distracting.